js/packages/web/src/views/auctionCreate/index.tsx

```diff
@@ -494,7 +533,8 @@ export const AuctionCreateView = () => {
       console.error('failed tracking new_listing');
     }
 
-    setAuctionObj(_auctionObj);
+    setAuctionObj(auctionInfo);
+    gotoNextStep();
```

Comment:
Go to next in the createAuctionManager function instead of instead waiting so can capture errors and not show congratulations. Only go to next on success.